src/aws-waf-mcp-server/.python-version

```diff
@@ -0,0 +1 @@
+3.12
```

Comment:
I see a difference in `3.12` here and `3.11` in the instructions. Is there anything restricting moving both to `3.13`? I also see the project requiring `>=3.12`

Reply:
I updated it to 3.13